utils/connectivity/conn_http.c

```diff
@@ -27,9 +27,31 @@ void mbedtls_debug(void *ctx, int level, const char *file, int line, const char
   fflush((FILE *)ctx);
 }
 
+static status_t free_info_context(connect_info_t *const info) {
+  if (info->https) {
+    mbedtls_ssl_close_notify(info->ssl_ctx);
+    mbedtls_net_free(info->net_ctx);
+    mbedtls_entropy_free(info->entropy);
+    mbedtls_ctr_drbg_free(info->ctr_drbg);
+    mbedtls_ssl_free(info->ssl_ctx);
+    mbedtls_ssl_config_free(info->ssl_config);
+    mbedtls_x509_crt_free(info->cacert);
+    free(info->net_ctx);
+    free(info->ssl_ctx);
+    free(info->ssl_config);
+    free(info->ctr_drbg);
+    free(info->entropy);
+    free(info->cacert);
```

Comment:
Should `free(info)` be called as well?

Reply:
I don't think so. `info` may be allocated on stack.